Sources/RealModule/Angle.swift

```diff
@@ -160,9 +160,32 @@ public extension Angle {
     }
 }
 
-extension Angle: Equatable {
-    public static func == (lhs: Angle<T>, rhs: Angle<T>) -> Bool {
-        lhs.radians.isApproximatelyEqual(to: rhs.radians)
+public extension Angle {
+    /// Checks whether the current angle is contained within a given closed range.
+    ///
+    /// - Parameters:
+    ///
+    ///     - range: The closed angular range within which containment is checked.
+    func contained(in range: ClosedRange<Angle<T>>) -> Bool {
```

Comment:
What is the value of these APIs over what's written in the body? I would suggest not including these separately.

What @NevinBR is getting at, I think, is that 361 degrees is contained "between" 0 degrees and 2 degrees. This does not offer that functionality.

For API naming purposes, incidentally, the function would need to be named `isContained`; and for style purposes, the access modifier `public` should be specified on each member (and therefore is not necessary on the extension itself).

Reply:
> What @NevinBR is getting at, I think, is that 361 degrees is contained "between" 0 degrees and 2 degrees. This does not offer that functionality.

Additionally, 175º and -175º are both “between” 170º and -170º (but they are **not** between -170º and 170º).

Reply:
Could @NevinBR shed some light on what the intention was please? Thank you

Reply:
I’m not entirely sure how normalization should come into play. *Probably* yes normalized angles should be used for comparison / containment, and if someone wants to work with the unnormalized value they can extract it and do so manually.

However, several of your examples do not behave as I would expect:

> 175º -> normalized to 175º -> contained in -170º...170º
-175º -> normalized to -175º -> contained in -170º...170º
175º -> normalized to 175º -> not contained in 170º...-170º
-175º -> normalized to -175º -> not contained in 170º...-170º

These should be:

> 175º -> normalized to 175º -> **not** contained in -170º...170º
-175º -> normalized to -175º -> **not** contained in -170º...170º
175º -> normalized to 175º -> **yes** contained in 170º...-170º
-175º -> normalized to -175º -> **yes** contained in 170º...-170º